test/addons/hello-world-esm/test.mjs

```diff
@@ -1,7 +1,6 @@
-// Flags: --experimental-modules
 /* eslint-disable required-modules */
 
 import assert from 'assert';
-import binding from '../addons/hello-world/build/Release/binding.node';
+import binding from './build/Release/binding.node';
```

Comment:
This still uses `build/Release`… did you verify that this fixes #16155?

Reply:
Whoops, I'll fix that. (No, I didn't verify. In my defense, I said "should now work in Debug builds" rather than "definitely does now work in Debug builds". :-D )

Reply:
Fixed! Should now work in Debug builds for really realz, although I haven't verified.